examples/hpc-enterprise-slurm.yaml

```diff
@@ -255,10 +255,41 @@ deployment_groups:
       # https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/develop/community/modules/compute/schedmd-slurm-gcp-v5-partition#compute-vm-zone-policies
       zones: $(vars.gpu_zones)
 
+  - id: h3_node_group
+    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    settings:
+      node_count_dynamic_max: 16
+      machine_type: h3-standard-88
+      instance_image:
+        family: $(vars.family)
+        project: $(vars.project)
+      service_account:
+        email: $(compute_sa.service_account_email)
+        scopes:
+        - https://www.googleapis.com/auth/cloud-platform
+      disk_type: pd-balanced # H3 does not support pd-ssd and pd-standard
```

Comment:
what other type of PD is supported?

Reply:
`pd-balanced` only, see https://cloud.google.com/compute/docs/compute-optimized-machines#h3_disks

Reply:
please add the link to the blueprint.